### ISSUE TITLE
Confusing pause symbol

### ISSUE THREAD

**Issue by RichardGG** [YclepticStudios/pebble-timer-plus#1]
While the animation looks cool. It's quite confusing.

Previously that color is used to highlight things, so that meaning should be consistent.

It changed from a foreground element(highlighting digits) to a background one(irrelevant to digits). But people will think it's still a foreground element.

I suggest moving the pause symbol somewhere else or transforming the circle instead of the highlight.

**Comment by YclepticStudios** [comment]
Thanks for the feedback. That is an issue... The pause symbol provides a convenient place to "put" the selection highlight, but it does so at the expense of "cleanness".

If you have any other suggestions on how to improve this, let me know.

**Comment by RichardGG** [comment]
Hmmm... Well to keep the highlight on screen and keep it's meaning, it should highlight the pause button.

Out of the elements on screen you could replace, the title probably has the least impact. So the name would turn into controls and the highlight would indicate that it's the current control.

![image](https://cloud.githubusercontent.com/assets/5629162/11053087/b9d76366-87a9-11e5-9557-7992424e0336.png)

BTW. This update is fantastic. You did a great job. The new convenient way to add hours makes it viable for me to replace alarm and I think it's so much more intuitive than before. Pause was the only thing that confused me.

**Comment by YclepticStudios** [comment]
Cool! I am considering simply having the selection layer zip off the screen... I would rather have it represent something useful on the screen, but I can't really think of a way to do that and have it still look nice. I kinda like the idea of having it at the top, but then it doesn't look very balanced to me... Kinda top heavy while missing text up there.

Here is a PBW of my current idea... I also changed it to green (some people thought it looked pinkish), and made a few other minor improvements. 

It wouldn't let me attach a pbw, so I changed the extension to .txt, just change it back to .pbw when you download.
[Timer+.txt](https://github.com/YclepticStudios/pebble-timer-plus/files/31146/Timer.txt)

Edit: I have now published this version.
